x-pack/packages/kbn-elastic-assistant/impl/assistant/chat_send/use_chat_send.tsx

```diff
@@ -136,6 +139,9 @@ export const useChatSend = ({
       assistantTelemetry?.reportAssistantMessageSent({
```

Comment:
Right, TBH I want to remove the duplicates but this will mess with historical data, message use will go down by half. Do you think thats ok? What do you think @peluja1012 ?

Reply:
Hi @stephmilovic, I wasn't aware we were counting "assistant" messages as well as "user" messages. Are we also counting serverside messages in the agent loop with EBT?

Reply:
@peluja1012  yes, we have a client side event counting each message sent, whether by user or assistant. it ends up being 1:1, which is kind of silly looking back. the server side event only produces one event per message sent by the user, as it is an api invocation measure

The question is, should we change our client side count to match up with the server side count? The downside would be usage from release to release, if tracked by AssistantMessageSent (client side event), would appear go down by half.